lib/cloud_controller/db.rb

```diff
@@ -139,6 +139,12 @@ def empty_from_sql
   end
 end
 
+class Sequel::Migration
+  def logger
```

Comment:
This is not used in the test migration.

Ideally one could use `log_info` from the `Sequel::Database` class in a migration ([here](https://github.com/jeremyevans/sequel/blob/70bcc08e47b060957e37584ee85c304d5698ef7c/lib/sequel/database/logging.rb#L31)).